docs/pages/setup/deployments/aws-terraform.mdx

```diff
@@ -467,7 +467,7 @@ $ ssh -i ${TF_VAR_key_name}.pem -o ProxyCommand="ssh -i ${TF_VAR_key_name}.pem -
 
 ```code
 # From EC2 host
-$ sudo tctl users add teleport-admin --roles=admin
+$ sudo tctl users add teleport-admin --roles=editor,access --roles=root
```

Comment:
Should the second `--roles` flag in this command be `--logins`?

Reply:
fixed